test/optimization/test_readme_sample.py

```diff
@@ -44,8 +44,8 @@ def print(*args):
         from qiskit.aqua import aqua_globals, QuantumInstance
         from qiskit.aqua.algorithms import QAOA
         from qiskit.aqua.components.optimizers import SPSA
-        from qiskit.optimization.ising import docplex, max_cut
-        from qiskit.optimization.ising.common import sample_most_likely
+        from qiskit.optimization.applications.ising import docplex, max_cut
```

Comment:
We need to make sure the Optimization sample code in that section of the README is updated to match this (since this test is to make sure that code from the README continues to run)